draft-ietf-webtrans-overview.md

```diff
@@ -222,7 +222,30 @@ attacks without an explicit indication from the client.
 All transport protocols MUST provide datagrams, unidirectional and
 bidirectional streams in order to make the transport protocols interchangeable.
 
-## Datagrams
+## Session-Wide Features  {#features-session}
+
+Any WebTransport protocol SHALL provide the following operations on the
+session:
+
+establish a session
+: Create a new WebTransport session given a URI {{!RFC3986}} and the origin
+  {{!RFC6454}}.
+
+terminate a session
+: Terminate the session while communicating to the peer an unsigned 32-bit
+  error code and an error reason string of at most 1024 bytes.  The error code
+  and string are optional; the default values are 0 and "".
```

Comment:
I might be misreading but H3 doesn't define any error code 0.

Reply:
The way this looks in practice is that if an H/3 session receives a FIN with no session close capsule, the API will return a `(0, "")` error, see [here](https://w3c.github.io/webtransport/#web-transport-termination) and [here](https://w3c.github.io/webtransport/#dictdef-webtransportcloseinfo).  We may want to update H/3 spec to mention those are semantically equivalent operations.